src/views/AppSidebar.vue

```diff
@@ -336,7 +336,7 @@ export default {
 				return this.task.createdMoment.calendar(null, {
 					lastDay: this.$t('tasks', '[Created yesterday at] LT'),
 					sameDay: this.$t('tasks', '[Created today at] LT'),
-					nextDay: this.$t('tasks', '[Created tomorrow at] LT'),
+					nextDay: this.$t('tasks', '[Created] L'),
 					lastWeek: this.$t('tasks', '[Created last] dddd [at] LT'),
 					nextWeek: this.$t('tasks', '[Created] dddd [at] LT'),
```

Comment:
More comments arrive at Transifex regarding future completeness.

Can we use a more simple way for "nextWeek" like '[Created] L' as well?

Reply:
Fine with me, but `[Created] dddd [at] LT` is not really specific to a certain tense, isn't it? It just write out as `Created Monday at 12:34:56`.